src/java_bytecode/java_bytecode_convert_class.cpp

```diff
@@ -385,6 +385,16 @@ void java_bytecode_convert_classt::convert(
       "."+id2string(f.name);
     new_symbol.mode=ID_java;
     new_symbol.is_type=false;
+
+    if(f.is_public)
+      new_symbol.type.set(ID_C_access, ID_public);
```

Comment:
Why is `ID_C_access` used here, whereas for classes and methods we use `ID_access`?

Reply:
Added comment -- it's so we don't get type clashes against constants, local variables, and anything else that might be assigned to a static field but doesn't itself carry a visibility modifier.